config/locales/pages/en.yml

```diff
@@ -4,11 +4,51 @@ en:
       header: The page you were looking for doesn't exist.
       body: You might want to double-check your link and try again. (404)
     privacy_policy:
-      privacy_act: >
-        Lorem ipsum dolor sit amet Lorem ipsum dolor sit amet Lorem ipsum dolor.
-        Lorem ipsum dolor sit amet Lorem ipsum dolor sit amet Lorem ipsum dolor.
-        Lorem ipsum dolor sit amet Lorem ipsum dolor sit amet Lorem ipsum dolor.
-      security_consent: >
-        Lorem ipsum dolor sit amet Lorem ipsum dolor sit amet Lorem ipsum dolor.
-        Lorem ipsum dolor sit amet Lorem ipsum dolor sit amet Lorem ipsum dolor.
-        Lorem ipsum dolor sit amet Lorem ipsum dolor sit amet Lorem ipsum dolor.
+      by_using: >
```

Comment:
I think in some other parts of the app we are using `paragraph_1` etc to describe different parts of a multi-part content page. While I like this naming convention for its explicitness, I do worry that as the content changes we will forget to change the keys and then they will be more confusing than helpful.

What do you think? Have you used this locale naming convention in the past?